package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "outdated": "npm outdated --depth 0",
     "preinstall": "./scripts/check_gm.sh",
-    "shrinkwrap": "npmshrink",
+    "shrinkwrap": "yarn install",
```

Comment:
Yarn will read the `package.json`, check if any of the dependencies have been updated, and install them if so. It finishes by updating the `yarn.lock`. More at https://yarnpkg.com/en/docs/cli/install

Actually, we wouldn't want to update the lock file in bulk any more, except for occasions where bulk updating dependencies is actually your goal. Instead, we would use `yarn upgrade package@version`, which will update the `package.json` and the `yarn.lock`, but only for that package (and it's dependencies).